docs/src/integrations/source/bitbucket.md

```diff
@@ -133,9 +133,10 @@ even if specifically set to true.
 
 ## Clones and commits
 
-When you run `platform get <projectID>` or use the clone command shown in the "Git" dropdown in the console to clone the project,
-you are actually cloning from your remote integrated repository,
-so long as you have the [appropriate access to do so](/administration/users.md#user-access-and-integrations). 
+You can clone your codebase by going to Console > Code > GIT and using the git clone command, or you can run `platform get <projectID>` with the CLI.
```

Comment:
I'm not sure that the change to a how to was completely necessary. I think it worked as a reference, it just needed a change to the drop-down. Haven't looked at the articles in depth though. :shrug: 

If sticking with this way here and below, I'd suggest a slightly different wording:

```suggestion
You can clone your codebase by running `platform get <projectID>` or in your project in the console going to Code > Git and running the `git clone` command.
```

The CLI way seems easier and in keeping with the rest of the article, though you can make that second if you strongly prefer.

But given that "Console" isn't something that appears in the UI like the other items in the list, I wouldn't have them next to each other like that.